modules/rosa-sharing-vpc-cluster-creation.adoc

```diff
@@ -0,0 +1,22 @@
+// Module included in the following assemblies:
+//
+// * networking/rosa-shared-vpc-config.adoc
+:_content-type: PROCEDURE
+[id="rosa-sharing-vpc-cluster-creation_{context}"]
+= Creating your cluster in a shared VPC
+
+The *cluster-creating AWS account* user runs the following command in their terminal to create the cluster in the shared VPC.
+
+[source,terminal]
+----
+rosa create cluster --cluster-name <cluster_name> --sts --operator-roles-prefix <prefix> --oidc-config-id <oidc_config_id> --region us-east-1 --subnet-ids <subnet_ids> --private-hosted-zone-id <hosted_zone_ID> --shared-vpc-role-arn <vpc-role-arn> --base-domain <dns-domain>
+----
+
+[IMPORTANT]
+====
+After deleting a shared-VPC cluster, three DNS records would not be removed from the private hosted zone.
```

Comment:
Should this read "will" instead of "would"? The past tense is confusing here.

Reply:
@wgordon17 - if I am understanding this issue correctly, these records would sometimes stick so we need to make this note more "this situation is possible."

OR is it a situation where the records never fully delete regardless what you do?